Packages/com.unity.multiplayer.samples.coop/Utilities/Net/UnityRelayUtilities.cs

```diff
@@ -62,7 +64,8 @@ public static async
             Debug.Log($"host: {allocation.HostConnectionData[0]} {allocation.HostConnectionData[1]}");
             Debug.Log($"client: {allocation.AllocationId}");
 
-            return (allocation.RelayServer.IpV4, (ushort)allocation.RelayServer.Port, allocation.AllocationIdBytes,
+            var dtlsEndpoint = allocation.ServerEndpoints.First(e => e.Secure);
```

Comment:
Check for `"dtls"` here as well.

Reply:
[680588e](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/485/commits/680588e09f045f797ed8706a19d8e6bf1038dda1)